### PR TITLE
Fix contract examples package json

### PR DESCRIPTION
**Pull request by ceyonur**
Fixes NPM package issue for imports
allows customized CMD to be run for e2e tests.
renames contract-examples to contracts

### CODE REVIEW

**Review by iknowhtml — APPROVED** [review]

**Review by richardpringle — APPROVED** [review]

File: tests/utils/subnet.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ import (
 	"github.com/onsi/gomega"
 )
 
-func RunHardhatTests(test string, rpcURI string) {
+func RunTestCMD(testCMD *exec.Cmd, rpcURI string) {
```
Comment (aaronbuchwald) [inline tests/utils/subnet.go:26]: Does this need to be external or do we only use `RunDefaultHardHatTests` ? If exported, could we add a comment?

**Review by aaronbuchwald — APPROVED** [review]
There's one comment I think we should address before merging, but other than that this LGTM

File: tests/utils/subnet.go (hunk 1 above)
Comment (ceyonur) [reply tests/utils/subnet.go:28]: `RunTestCMD` could be used if paths and test commands are different. Developers can customize their commands to run different tests (not only hardhat) in different networks (not only in local). I added more comments.

**Review by aaronbuchwald — APPROVED** [review]

**Comment by richardpringle** [comment]
🎉

**Pull request merged**